app/src/main/java/io/homeassistant/companion/android/notifications/NotificationActionReceiver.kt

```diff
@@ -42,32 +45,46 @@ class NotificationActionReceiver : BroadcastReceiver() {
             return
         }
 
-        when (intent.action) {
-            FIRE_EVENT -> fireEvent(notificationAction)
-            OPEN_URI -> openUri(context, notificationAction)
-        }
-
         val messageId = intent.getIntExtra(EXTRA_NOTIFICATION_ID, -1)
-        if (messageId != -1) {
-            val notificationService: NotificationManager =
-                context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
-            notificationService.cancel(messageId)
+        val onComplete: () -> Unit = {
+            (context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager)
+                .cancel(messageId)
+        }
+        val onFailure: () -> Unit = {
+            Handler(context.mainLooper).post {
+                Toast.makeText(context, "Unable to send event to HA.", Toast.LENGTH_LONG).show()
```

Comment:
I feel like this text can be better. 

Maybe "Failed to communicate with Home Assistant" ?

Reply:
Oh wait this is only for events when clicking notification actions.

Reply:
This is ok .

Reply:
This should've been a localizable string :(

Reply:
Never mind, it ended up being a proper string!